fiesta/apps/events/migrations/0005_alter_pricevariant_amount_currency.py

```
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.28 on 2026-03-07 20:56
+
+from django.db import migrations
+import djmoney.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('events', '0004_alter_pricevariant_amount_currency'),
+    ]
```

Ruff (0.15.4)

[warning] Mutable default value for class attribute

(RUF012)